src/concrete/ml/sklearn/linear_model.py

```diff
@@ -347,7 +347,7 @@ def _get_training_quantized_module(
         # Enable the underlying FHE circuit to be composed with itself
         # This feature is used in order to be able to iterate in the clear n times without having
         # to encrypt/decrypt the weight/bias values between each loop
-        configuration = Configuration(composable=True)
+        configuration = Configuration(composable=True, compress_evaluation_keys=True)
```

Comment:
compress pbs keys for training

Reply:
which makes me thinkg, it should be using `enable_input_compression` as well here !

Reply:
yes, you're right, same as for DF which was not enabled before

Reply:
actually same as below, we should probably revert this change : it's emabled by QM's compilation anyway